department-of-veterans-affairs/va.gov-cms
# VACMS-7420 configure claro footer menu

## Description

Closes #7420 .

## Testing done
Added links to the footer menu and saw them change on the login screen.

## Screenshots
Existing footer menu with new links added:
![image](https://user-images.githubusercontent.com/8375335/147983177-1906589e-20be-43f7-86d2-18e595cd87fd.png)

Frontend login screen showing new menu-based links:
![image](https://user-images.githubusercontent.com/8375335/147983119-65fd9dea-3501-4603-bd2d-6973fc2689bc.png)


## QA steps

As a cms admin I can
1. Edit the menu items at `/admin/structure/menu/manage/footer` 
2. Log out and see that they're updated on the login screen
1. Then validate Acceptance Criteria from issue
- [x] Footer Menu Has been created
- [x] Workbench menu access set to "VA.gov CMS Knowledge Base"

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
